README.md

```diff
@@ -1,6 +1,6 @@
 # Movies list - Fetch movies
 - Replace `<your_account>` with your Github username in the
- [DEMO LINK](https://<your_account>.github.io/react_movies-list-fetch-movies/)
+ [DEMO LINK](https://Ol-Lav.github.io/react_movies-list-fetch-movies/)
```

Comment:
https://user-images.githubusercontent.com/38065883/167926652-56c3e4fd-669d-48fb-82d4-fc24f198a2ad.mov

For now it's not working